src/staging/staging_main.py

```diff
@@ -143,14 +143,24 @@ def run_staging(
 
         # Write both snapshots to feather file at given path
         if is_network:
-            helpers.write_snapshot_to_feather(
+            feather_fname = f"{snapshot_name}_corrected.feather"
+            s_feather_fname = f"{secondary_snapshot_name}.feather"
+            helpers.df_to_feather(
                 feather_path,
-                snapshot_name,
+                feather_fname,
                 full_responses,
-                write_feather,
-                secondary_snapshot_name,
-                secondary_full_responses,
+                write_feather
+
             )
+            if secondary_full_responses is not None:
+                helpers.df_to_feather(
+                    feather_path,
+                    s_feather_fname,
+                    secondary_full_responses,
+                    write_feather
+
+                )
```

Comment:
When I run it with the secondary_full_responses it is erroring at src.staging.spp_snapshot_processing - full_responses - INFO:Starting Data Transmutation... with KeyError: 'Only a column name can be used for the key in a dtype mappings argument.'

Reply:
can you post the full error?

Reply:
I have fixed this by adding the instance col to the secondary snapshot (fix already in place for DevTest)